### PR TITLE
Update elastic-container-service.md

### PR DESCRIPTION
**Pull request by boofinka**
Updated to use the correct docker image: https://github.com/logzio/logzio-aws-ecs

# What changed

----

## Pages to review

- https://deploy-preview-525--logz-docs.netlify.app/shipping/log-sources/elastic-container-service.html

### CODE REVIEW

**Comment by shalper** [comment]
@boofinka This is being investigated by the team. More to follow.

**Comment by boofinka** [comment]
Ok

File: _source/logzio_collections/_log-sources/elastic-container-service.md [hunk 1]
```diff
@@ -60,15 +62,13 @@ by setting the `ignoreOlder` parameter of the new docker
...
 {:.info-box.note}
 
-| Parameter | Description |
+| Paramater | Details |
 |---|---|
-| LOGZIO_TOKEN <span class="required-param"></span> | Your Logz.io account token. {% include log-shipping/replace-vars.html token=true %} |
-| LOGZIO_URL <span class="required-param"></span> | Your Logz.io listener URL and port. {% include log-shipping/replace-vars.html listener=true %} |
- ignoreOlder <span class="default-param">`3h`</span>|  Set a time limit on back shipping logs. Upgrading to a newer version of docker-collector-logs while it is already running will cause it to resend logs that are within the `ignoreOlder` timeframe. You can minimize log duplicates by setting the `ignoreOlder` parameter of the new docker to a lower value (for example, `20m`). |
-{:.paramlist}
+| **LOGZIO_URL** | **Required**. Your Logz.io listener URL. Replace `<LISTENER-URL>` with your region's listener URL. For more information on finding your account's region, see [Account region](https://docs.logz.io/user-guide/accounts/account-region.html) in the Logz.io Docs. |
```
Comment (imnotashrimp) [inline _source/logzio_collections/_log-sources/elastic-container-service.md:67]: @shalper If this PR is indeed going through, these table rows need to match the docs format - with the 'default' and 'required' `<span>` tags, not-bolded parameter name, and using the Jekyll include.

File: _source/logzio_collections/_log-sources/elastic-container-service.md [hunk 2]
```diff
@@ -60,15 +62,13 @@ by setting the `ignoreOlder` parameter of the new docker
 to a lower value (for example, `20m`).
 {:.info-box.note}
 
-| Parameter | Description |
+| Paramater | Details |
```
Comment (imnotashrimp) [inline _source/logzio_collections/_log-sources/elastic-container-service.md:65]: This change isn't needed, and there's a typo anyway. Recommend reverting it.

File: _source/logzio_collections/_log-sources/elastic-container-service.md [hunk 3]
```diff
@@ -35,20 +36,21 @@ The Docker logs directory and docker.sock are mounted to the container, allowing
...
 ##### Run the Docker image
 
 For a complete list of options, see the parameters below the code block. 👇
 
 ```shell
-docker run -d --name=docker-collector-logs \
---env LOGZIO_TOKEN="<<SHIPPING-TOKEN>>" \
---env LOGZIO_URL="<<LISTENER-HOST>>:5015" \
+docker run --name=logzio-aws-ecs \
+-e "LOGZIO_URL_1=https://<LISTENER-URL>:8071?token=<ACCOUNT-TOKEN>" \
```
Comment (imnotashrimp) [inline _source/logzio_collections/_log-sources/elastic-container-service.md:48]:
```suggestion
-e "LOGZIO_URL_1=https://<<LISTENER-HOST>>:8071?token=<<ACCOUNT-TOKEN>>" \
```

- Should be listener host, not url
- vars need double angle brackets `<< >>`

**Comment by YajB** [comment]
changes should not be merged

**Pull request closed without merging**